src/utils.js

```diff
@@ -33,10 +33,7 @@ function validate(opts) {
 }
 
 function isRetriable(err) {
-  if (!err.response) {
-    return false;
-  }
-  const statusCode = err.response.statusCode;
+  const statusCode = err.response ? err.response.statusCode : err.body? err.body.status_code : 0;
```

Comment:
What was the situation in which there was no `err.response` but there was an `err.body.status_code`? Has that come from the changes you made in `NgrokClient`?

Reply:
From what I can tell, this doesn't come from changes to the NgrokClient. This change was made because I was receiving errors in the error handling that should have gone through the retry logic but didn't because of this problem.

Reply:
Given that one of the unit tests dealing with paid plans is broken now, it looks like this needs further investigation.

Reply:
The problem occurs when someone has a paid plan and it's in use on another machine. In other words, a tunnel cannot be created because only one tunnel is allowed.

Reply:
I'd put this in a separate PR personally